changelogs/head.asciidoc

```diff
@@ -6,7 +6,8 @@
 
 [float]
 ==== Added
-- Upgrade Go to 1.12.7 {pull}XXX[XXX].
+- Upgrade Go to 1.12.7 {pull}2483[2483].
```

Comment:
ok to change this here as backporting changelogs always involves some manual updates anyway